lib/Profile.dart

```diff
@@ -0,0 +1,227 @@
+                                top: _picDiameter * 0.66
+                            )
+                          ],
+                        )
+                    ),
+                    Padding(
+                      padding: EdgeInsets.only(bottom: 30),
+                      child: Stack (
+                        overflow: Overflow.visible,
+                        children: [
+                          Row(
+                            //crossAxisAlignment: CrossAxisAlignment.end,
+                              children: [
+                                Text("Terry Cruz",
+                                    style: TextStyle(
+                                      fontSize: 22,
+                                      fontWeight: FontWeight.bold,
+                                    )
+                                ),
+                                IconButton(
+                                  icon: Icon(
+                                      Icons.edit,
+                                      size: 20
+                                  ),
+                                  onPressed: () {
+                                  },
+                                )
+                              ]
+                          ),
+                          Positioned(
+                            child: Text("Joined 03/2020",
+                                style: TextStyle(
+                                  fontSize: 14,
+                                  color: Theme.of(context).accentColor,
+                                )
+                            ),
+                            top: 45,
+                          )
+                        ],
+                      )
+                    )
+
+                  ]
+              )
+          )
+        ]
+    );
+  }
+}
+
+class AccountInfo extends StatefulWidget {
+  @override
+  _AccountInfoState createState() => _AccountInfoState();
+}
+
+class _AccountInfoState extends State<AccountInfo> {
+
+  Widget infoRow(BuildContext context, IconData icon, String text) {
+    return Padding(
+        padding: EdgeInsets.only(top: 10, bottom: 10),
+        child: Row(
+          children: <Widget>[
+            Icon(icon),
+            SizedBox(width: 19),
+            Expanded (
+              child: Text(
+                text,
+                style: TextStyle(
+                  fontSize: 17,
+                  color: Theme.of(context).accentColor,
+                ),
+              ),
+            ),
+            IconButton(
+              icon: Icon(Icons.arrow_forward_ios),
+              onPressed: () {},
+            )
+          ],
+        )
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    final List<IconData> icons = new List();
+    icons.add(Icons.mail_outline);
+    icons.add(Icons.phone);
+    final List<String> tempText = new List();
+    tempText.add("terrycruz@hotmail.com");
+    tempText.add("Add your number");
```

Comment:
Why do it in this way? Can't you just initialize the list instead of initializing an empty one and adding to it? In this way, those lists will actually be "final"